Rancher 2.5.9 - backup / restore causes two default-tokens in each namespace (#4253)
rancher/docs: content/rancher/v2.5/en/backups/restoring-rancher/_index.md

This happens because one of the resources that was just restored has finalizers but the related resources have been deleted so the handler cannot find it.

To eliminate the errors, we need to find and delete the resource that causes the error. See more information [here](https://github.com/rancher/rancher/issues/35050#issuecomment-937968556)

> In Rancher 2.5, under certain circumstances after restoring the backup, Rancher creates two default-tokens in each namespace:

Reviewer: ```suggestion
Under certain circumstances after restoring the backup, Rancher creates two default-tokens in each namespace:
```

Reviewer: Since the observation is in 2.5 versioned docs, you can omit mentioning the version.

```
kubectl get secret -A|grep default-token
c-gt5hn default-token-jmck8 kubernetes.io/service-account-token 3 85m
c-gt5hn default-token-ss54l kubernetes.io/service-account-token 3 85m
c-plt7m default-token-qnwxt kubernetes.io/service-account-token 3 85m
c-plt7m default-token-v75f7 kubernetes.io/service-account-token 3 85m
```

> This has been fixed in Rancher 2.6.4. See more information [here]

Reviewer: ```suggestion
```

> (https://github.com/rancher/backup-restore-operator/issues/165#issuecomment-1079475984)

Reviewer: ```suggestion
```

> As a workound, you can use the following script, which removes the redundant restored default-token- secrets from restored Namespaces.

Reviewer: ```suggestion
As a workaround, use the following script which removes the redundant restored default-token- secrets from restored Namespaces:
```
